test/jdk/javax/swing/JComponent/7154030/bug7154030.java

```diff
@@ -81,25 +84,36 @@ public void run() {
                     button.setOpaque(true);
                     button.setVisible(false);
                     desktop.add(button);
+                    desktop.setMinimumSize(new Dimension(300, 300));
+                    desktop.setMaximumSize(new Dimension(300, 300));
 
                     frame.setContentPane(desktop);
-                    frame.setSize(300, 300);
+                    frame.setMinimumSize(new Dimension(350, 350));
+                    frame.setMaximumSize(new Dimension(350, 350));
+                    frame.pack();
                     frame.setLocationRelativeTo(null);
                     frame.setVisible(true);
                     frame.setDefaultCloseOperation(JFrame.DISPOSE_ON_CLOSE);
                 }
             });
 
-            robot.delay(1000);
             robot.waitForIdle(1000);
+            robot.delay(1000);
 
+            Dimension screenSize = Toolkit.getDefaultToolkit().getScreenSize();
+            Rectangle screen = new Rectangle(0, 0, (int) screenSize.getWidth(), (int) screenSize.getHeight());
             Rectangle bounds = frame.getBounds();
             Insets insets = frame.getInsets();
             locx = bounds.x + insets.left;
             locy = bounds.y + insets.top;
             frw = bounds.width - insets.left - insets.right;
             frh = bounds.height - insets.top - insets.bottom;
 
+            imageInit = robot.createScreenCapture(new Rectangle(locx, locy, frw, frh));
```

Comment:
Looks like the result of this screen capture is unused.

Reply:
Yes, accidentally copied line instead of moving it.